streams-tests/shared/src/test/scala/zio/stream/ZStreamSpec.scala

```diff
@@ -2792,7 +2792,28 @@ object ZStreamSpec extends ZIOBaseSpec {
                 _     <- f.join
               } yield assertTrue(count == 0)
             }
-          } @@ TestAspect.jvmOnly @@ jvm(nonFlaky(20))
+          } @@ TestAspect.jvmOnly @@ jvm(nonFlaky(20)),
+          test("accumulates parallel errors") {
```

Comment:
Sorry I wasn't very clear in my previous comment. What I meant was, would it be possible to add a test to show that when we are processing a ZStream in parallel using the `*Unordered` method, faster executing effects will be emitted first?